fastapi_pagination/ext/ormar.py

```diff
@@ -0,0 +1,23 @@
+from typing import Optional, Type, Union
+
+from ormar import Model, QuerySet
+
+from ..api import create_page, resolve_params
+from ..bases import AbstractPage, AbstractParams
+
+
+async def paginate(query: Union[QuerySet, Type[Model]], params: Optional[AbstractParams] = None) -> AbstractPage:
+    if not isinstance(query, QuerySet):
```

Comment:
Please, add tests to cover case when query is a model instance.

Example for `gino`:
https://github.com/uriyyo/fastapi-pagination/blob/25bcbb9fe101886f0481472bf352876f9c9e6517/tests/ext/test_gino.py#L35-L44